scripts/insert_into_appointments.sql

```diff
@@ -1 +1,7 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(date,time,comment,patient_id,doctor_id) VALUES
+  ('2016-08-12', '10:00 AM','Marsha Wilcox seeing Dr. Vicky Hilton',8, 4),
+  ('2016-08-17', '1:00 PM','Ivory Talley seeing Dr. Spencer Hubbard',5, 6),
+  ('2016-08-12', '10:00 AM','Marsha Wilcox seeing Dr. Spencer Hubbard',8, 6),
+  ('2016-08-17', '1:00 PM','Marsha Wilcox seeing Dr. Vicky Hilton',7, 4);
```

Comment:
This would only work if we knew the ID's ahead of time. Here's a query you can use:

```sql
-- insert appointments creating connections between doctors and patients
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT DATE '2017-08-12', TIME '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hilton'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Talley')
  UNION
  SELECT '2017-08-17', '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
   (SELECT id FROM doctors WHERE family_name = 'Hilton'),
   (SELECT id FROM patients WHERE family_name = 'Hodge')
;
```